app/src/main/java/com/example/trainingintive/util/ExceptionExtensions.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.trainingintive.util
+
+import com.example.trainingintive.R
+import java.net.UnknownHostException
+
+fun Exception.toErrorTextId() =
```

Comment:
It could be an extension to `Throwable`, this way you will avoid the need of casting in VW :)